public/app/features/dashboard/dashgrid/PanelChromeAngular.tsx

```diff
@@ -90,20 +92,8 @@ export class PanelChromeAngularUnconnected extends PureComponent<Props, State> {
   }
 
   onPanelRenderEvent = (payload?: any) => {
-    const { alertState } = this.state;
-
-    if (payload && payload.alertState && this.props.panel.alert) {
-      this.setState({ alertState: payload.alertState });
-    } else if (payload && payload.alertState && !this.props.panel.alert) {
-      // when user deletes alert in panel editor the source panel needs to refresh as this is in the mutable state and
-      // will not automatically re render
-      this.setState({ alertState: undefined });
-    } else if (payload && alertState) {
-      this.setState({ alertState: undefined });
-    } else {
-      // only needed for detecting title updates right now fix before 7.0
-      this.forceUpdate();
-    }
+    // only needed for detecting title updates right now fix before 7.0
+    this.forceUpdate();
```

Comment:
Not at all sure about this one, don't know the consequences of doing this without conditions.